.gitattributes

```diff
@@ -1,3 +1,3 @@
 cassettes/* linguist-generated=true
 factory_data/* linguist-generated=true
-flow-typed/* linguist-generated=true
+frontends/api/src/generated/* linguist-generated=true
```

Comment:
I didn't mark `openapi.yaml` as generated (even though it is) because, IMO, it is useful to see its diffs in PRs